src/client/eppo-client.ts

```diff
@@ -167,18 +167,21 @@ export default class EppoClient implements IEppoClient {
   private assignmentLogger: IAssignmentLogger | undefined;
   private isGracefulFailureMode = true;
   private assignmentCache: AssignmentCache<Cacheable> | undefined;
-  private configurationStore: IConfigurationStore<Flag | ObfuscatedFlag>;
+  private flagConfigurationStore: IConfigurationStore<Flag | ObfuscatedFlag>;
+  private banditConfigurationStore: IConfigurationStore<BanditParameters> | undefined;
```

Comment:
`EppoClient`'s existing store is renamed to `flagConfigurationStore` and we've now added a `banditConfigurationStore`.

This parallels the two-store approach done in our [Java](https://github.com/Eppo-exp/java-server-sdk/blob/main/src/main/java/com/eppo/sdk/EppoClient.java#L549) and [Python](https://github.com/Eppo-exp/python-sdk/blob/main/eppo_client/configuration_requestor.py#L19) SDKs.

Reply:
Is the plan to keep `undefined` as an option (for client side SDKs)? Or remove it in a future PR?

Reply:
We'll keep it because if were punting on bandits for client SDKs, then the js client SDK will pass in `undefined` to effectively "disable" bandits. However, customers will never be passing in this argument--just our upstream SDKs